README.md

```diff
@@ -15,9 +15,9 @@ Visit:
 
 To run docs locally:
 
-1. Clone this repo
-2. Run `npm install`
-3. Run `npm start` inside the `websites` folder.
+1. Clone this repo,
+2. Run `npm install` inside the `websites` folder,
+3. Run `npm start` also inside the `websites` folder.
```

Comment:
```suggestion
1. Clone this repo
2. Navigate to `websites` folder inside the repo
2. Run `npm install`
3. Run `npm start`
```

This would be even clearer

Reply:
This is better, yeah